Fix typedef redefinition for struct env_md_ctx_st to unbreak with LibreSSL

Index: src/crypto/Digest.h
--- src/crypto/Digest.h.orig
+++ src/crypto/Digest.h
@@ -57,7 +57,6 @@
 #define URI_ECDSA_SHA512 "http://www.w3.org/2001/04/xmldsig-more#ecdsa-sha512"
 
 #ifdef LIBRESSL_VERSION_NUMBER
-using EVP_MD_CTX = struct env_md_ctx_st;
 #else
 using EVP_MD_CTX = struct evp_md_ctx_st;
 #endif
